html/changelogs/fluffyghost-botsnavbeaconspath.yml

```
@@ -0,0 +1,49 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes:
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: FluffyGhost
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "Fixed cleanbots so they detect flour."
+  - bugfix: "Potentially fixed farmbots to not turn the server's CPU into molten metal."
+  - bugfix: "Fixed navbeacons, they only work on the same Z-level."
+  - refactor: "Refactored our bots to use TG's JPS (Jump Point Search) instead of AStar, which is more efficient."
+  - rscadd: "Navbeacons can now be print by the roboticist."
```

```suggestion
  - rscadd: "Navbeacons can now be printed in protolathes."
```
More accurate, Scientist in RnD can alos make Navbeacons with the changes in this PR (however unlikely it is to get a scientist to be fair)